Vectorize: Add Content to the Best Practices Documentation (#21913)
cloudflare/cloudflare-docs: src/content/docs/vectorize/reference/metadata-filtering.mdx

```

#### Range query involving strings

> Range queries can be used to implement **prefix searching** on string metadata fields. This also like a **starts_with** filter.

Reviewer: ```suggestion
Range queries can implement **prefix searching** on string metadata fields. This is also like a **starts_with** filter.
```
